driver/chardev.c

```diff
@@ -76,9 +78,9 @@ static int vck_open(struct inode *, struct file *);
 static int vck_release(struct inode *, struct file *);
 static int vck_mmap(struct file *, struct vm_area_struct *);
 
-static int vck5000_ioctl_get_version(struct file *filep, void *data);
-static int vck5000_ioctl_create_queue(struct file *filep, void *data);
-static int vck5000_ioctl_destroy_queue(struct file *filp, void *data);
+static int amdair_ioctl_get_version(struct file *filep, void *data);
```

Comment:
I'm moving in that direction, but slowly on this point because it's still not clear to me which functions are generic, and which ones are vck5000 specific.